Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -6268,9 +6268,17 @@ def is_demisto_version_ge(version, build_number=''):
     """
     try:
         server_version = get_demisto_version()
-        return \
-            server_version.get('version') >= version and \
-            (not build_number or int(server_version.get('buildNumber')) >= int(build_number))
+        if server_version.get('version') > version:
+            return True
+        elif server_version.get('version') == version:
+            if build_number:
+                return int(server_version.get('buildNumber')) >= int(build_number)
+            return True  # No build number
+        else:
+            return False
+    except ValueError:
+        # dev editions are not comparable
```

Comment:
Let's add an info log here. It should only happen on dev, so if we do reach this flow it is worth logging.